Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md

```diff
@@ -0,0 +1,78 @@
+
+## Step 1: Deploy this logic app to your organization
+
+If your Azure environment meets the prerequisites, and you're familiar with using Azure Resource Manager templates, these steps help you sign in directly to Azure and open the Azure Resource Manager template in the Azure portal. For more information, see the article, [Deploy resources with Azure Resource Manager templates and Azure portal](https://docs.microsoft.com/azure/azure-resource-manager/templates/overview).
+
+Select the following image to sign in with your Azure account and open the logic app in the Azure portal:
+
+   [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FPut-MDEAlert-Hunting-GitHub%2Fazuredeploy.json)         [![Deploy to Azure Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FPut-MDEAlert-Hunting-GitHub%2Fazuredeploy.json)
+   
+
+
+1. In the portal, on the **Custom deployment** page, enter or select these values:
+
+   | Property | Value | Description |
+   |----------|-------|-------------|
+   | **Subscription** | <*Azure-subscription-name*> | The name for the Azure subscription to use |
+   | **Resource group** | <*Azure-resource-group-name*> | The name for a new or existing Azure resource group |
+   | **Region** |  <*Azure-region-for-all-resources*> | The Azure region to use for all resources, if different from the default value. This example uses the default value, `[resourceGroup().location]`, which is the resource group location. |
+   | **Playbook Name** | <*Put-MDEAlert-Hunting-GitHub*> | The name of the playbook to deploy.|
+   | **GitHub Repo Name** | <*GitHubRepoName*> | The GitHub Repository Name that's connected to Sentinel.|
+   | **GitHub Repo Owner Name** | <*GitHubRepoOwnerName*> | The GitHub Owner Name associated with the GitHub repo connected to Sentinel. |
+   | **Key Vault GitHub Secret URL** | <*KeyVaultGitHubCredentialsURL*> | The URL for your GitHub Personal Access token from Azure Key Vault|
+
+1. When you're done, select **Review + Create** and finally **Create**.
+
+## Step 2: Add your Logic App System Managed Identity to Key Vault with the right permissions
+
+This logic app uses a System Managed Identity to access secrets from Key Vault. As a prerequisite you must have a Key Vault created with your GitHub API Key stored and give this Logic App's System Managed Identity the ability to Read the associated Secret at the Key Vault Git Hub Credentials URL. To learn more about how to use managed identities within Logic Apps, see the article [**Logic Apps and Managed Identities**](https://docs.microsoft.com/azure/logic-apps/create-managed-service-identity).
+
```

Comment:
Steps are looking good, 
But from customer point of view and large audience, I recommend if you can tell them straight on what to do related to key vault access policies, 
Like mentioned below:

A. Assign access policy on key vault for Playbook to fetch the secret key

- Select the Keyvault resource where you have stored the secret
- Click on Access policies Blade
- Click on Create
- Under Secret permissions columun , Select Get , List from "Secret Management Operations"
- Click next to go to Principal tab and choose your deployed playbook name
- Click Next leave application tab as it is.
- Click Review and create
- Click Create

Reply:
@SCStelz  , Any update on this